apps/web/src/utils/calls/farms.ts

```diff
@@ -52,7 +52,7 @@ export const unstakeFarm = async (
   gasLimit?: bigint,
 ) => {
   const value = new BigNumber(amount).times(DEFAULT_TOKEN_DECIMAL).toString()
-
+  logGTMClickUnStakeFarmEvent()
```

Comment:
let's move this line after the if condition, and log right before interacting with sc